crates/propolis-api-types/src/lib.rs

```diff
@@ -186,6 +194,13 @@ pub struct InstanceProperties {
     pub vcpus: u8,
 }
 
+impl InstanceProperties {
+    /// Return the name of the VM as managed by the VMM.
+    pub fn vm_name(&self) -> String {
+        self.id.to_string()
```

Comment:
return the `&str`, and leave it up to the caller if they want to turn it into an owned string?

Reply:
`InstanceProperties::id` is a `Uuid`, so we have to return either the ID itself or an owned string, I think?

Reply:
Ah, oversight on my part.